DBM-Naxx/ConstructQuarter/Patchwerk.lua

```diff
@@ -3,20 +3,27 @@ local L		= mod:GetLocalizedStrings()
 
 mod:SetRevision(("$Revision: 2869 $"):sub(12, -3))
 mod:SetCreatureID(16028)
+mod:SetUsedIcons(8)
```

Comment:
Lets use more emotes, since if healer will be slacking there prolly will be more debuffs than one at a time, plus we already are using table method anyway

Reply:
How do i use more emotes? just increase SetUsedICons?

Reply:
Ye you just chande it from just using 8 emote, to for example (5, 6, 7, 8) and that will use 4 emotes instead

Reply:
ah now i got it :)
probably wont be used since Grievous Throw has 1min CD, if you have more than 1 buff active your doing it very wrong BUT its possible so i changed it, thanks! 👍